githubutils/repo.go

```diff
@@ -145,17 +145,100 @@ func GetRawGitFile(ctx context.Context, client *github.Client, content *github.R
 	return byt, err
 }
 
+// A RepoFilter can check if a release shold be filtered out based on a condition.
+//  In general filters may be stateful and be a StatefulRepoFilter
+type RepoFilter interface {
```

Comment:
How is this different that a RepositoryReleasePredicate? It has the same interface, so I don't quite follow how each are intended to be used

Reply:
You had stated that you wanted the LTS determining filter to exist in the same place as our existing predicates.

Predicate becomes a bad name for a repofiltering mechanism once you are no longer filtering purely on predicates.